spec/spec_helper.rb

```diff
@@ -10,6 +10,8 @@
 require 'rspec'
 require 'webmock/rspec'
 require 'psulib_traject'
+require 'rspec-sidekiq'
+require 'sidekiq/testing/inline'
```

Comment:
Put all of the changes to spec_helper into a file called `spec/support/sidekiq.rb`. It will be autoloaded from line 19. The spec_helper file is auto-generated when installing or updating rspec, so we minimize any additions to it to keep the diffs small.

Reply:
added.